purview/app.py

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+import os
+
+from flask import Flask
+from werkzeug.exceptions import default_exceptions
+
+from purview.logger import init_logging
+from purview.error import json_error, ValidationError
+from purview.api.v1 import api_v1
+
+
+def load_config(app):
+    default_config_file = None
+    if os.getenv('DEV') == 'true':
```

Comment:
This is fine but maybe we should do `if os.getenv('DEV') and os.getenv('DEV').lower() == 'true:` for ease.